components/brave_vpn/resources/panel/theme/vpn-light.ts

```diff
@@ -0,0 +1,23 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import ITheme from 'brave-ui/theme/theme-interface'
+import IThemeVPN from './vpn-theme'
+import defaultTheme from 'brave-ui/theme/brave-default'
+import colors from 'brave-ui/theme/colors'
+
+const walletLightTheme: ITheme & IThemeVPN = {
+  ...defaultTheme,
+  name: 'VPN Light',
+  color: {
+    ...defaultTheme.color,
+    text: colors.neutral900,
+    panelBackground: '#F9F9FD',
+    panelBackgroundSecondary: colors.neutral000,
+    outlineColor: 'rgba(255,80,0,0.2)'
```

Comment:
`IThemeVPN` only defines `outlineColor` yet we're defining values for a few other properties here - does that mean we're differentiating core theme values for some reason? This can all be considered for a follow-up.